### PR TITLE
T31561

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
Only dropped translations, they're conflicting.

### CODE REVIEW

**Comment by wjt** [comment]
- We lost 706386d52dceeff5cd4a8c188015c5b57c790aa5 as part of this rebase which adds a bunch of new languages. The way I normally fix the conflicts in patches like that is to reapply the `LINGUAS` bit then regenerate the `.po` files and commit the result. But we could do that separately. I'll go make a "sort out translations" ticket for switching Transifex over to the eos4 branch.
- b807b5701bd35557614f52925f489c965d279ee6 can go, per [this doc](https://docs.google.com/document/d/1MMT1HTD-rZPJ7k20TYD3zX3MslSK2X_uGPB6Hxle9Kc/)
- Sounds like everything else was either already upstream or applied cleanly?

**Comment by andrunko** [comment]
>     * We lost [706386d](https://github.com/endlessm/gnome-shell/commit/706386d52dceeff5cd4a8c188015c5b57c790aa5) as part of this rebase which adds a bunch of new languages. The way I normally fix the conflicts in patches like that is to reapply the `LINGUAS` bit then regenerate the `.po` files and commit the result. But we could do that separately. I'll go make a "sort out translations" ticket for switching Transifex over to the eos4 branch.
> 

Done that.

>     * [b807b57](https://github.com/endlessm/gnome-shell/commit/b807b5701bd35557614f52925f489c965d279ee6) can go, per [this doc](https://docs.google.com/document/d/1MMT1HTD-rZPJ7k20TYD3zX3MslSK2X_uGPB6Hxle9Kc/)
> 
Reverted that change, I plan to squash it while merging.

>     * Sounds like everything else was either already upstream or applied cleanly?
Looks like the case from what I see.

**Review by andrunko — APPROVED** [review]
LGTM, @wjt do you want to have a final look on latest changes before merging?

File: js/misc/eos-launch.js
Comment (wjt) [inline js/misc/eos-launch.js:50]: I had hoped that this wouldn't be needed any more. I searched `/usr` on my Endless OS system and found a single usage in eos-gates. https://phabricator.endlessm.com/T28309 was the last time we tried and failed to drop it; back then it was the exploration centre that needed it. IIRC the disco feed also used to use it.

Let's not block this PR on that but can we file an EOS-4.0 task to remove its last use and then get rid of it?

**Review by wjt — APPROVED** [review]
LGTM to squash and force-push. I have not reviewed #718, nor tested either.

**Comment by andrunko** [comment]
Landed manually.

**Pull request closed without merging**
